include/lsst/meas/base/PeakLikelihoodFlux.h

```diff
@@ -38,16 +38,16 @@ namespace meas {
 namespace base {
 
 /**
- *  @brief C++ control object for peak likelihood flux.
+ *  @brief C++ control object for peak likelihood instFlux.
```

Comment:
Consider using "instrument flux" instead of "instFlux" here and in most of the rest of the documentation. I think it would enhance clarity.